financial-connections/src/main/java/com/stripe/android/financialconnections/browser/BrowserManager.kt

```diff
@@ -0,0 +1,58 @@
+package com.stripe.android.financialconnections.browser
+
+import android.app.Application
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsIntent
+import javax.inject.Inject
+
+internal class BrowserManager @Inject constructor(
+    val context: Application
+) {
+
+    /**
+     * Returns `true` if there's an app that can open an http url.
+     */
+    fun canOpenHttpUrl(): Boolean = getPackageToHandleUri(Uri.parse("https://")) == null
```

Comment:
Should this be `canOpenHttpsUrl` instead of `canOpenHttpUrl` based in the method content?